.babelrc

```diff
@@ -1,3 +1,3 @@
-{
-  "presets": [ "es2015" ]
-}
+{
+  "presets": [ "es2015" ]
+}

```

Comment:
I'm generally against touching files if nothing changes to avoid the diff. Let's keep the original one.